usr/sbin/registercloudguest

```diff
@@ -833,7 +833,7 @@ def main(args):
         logging.error(err_msg % (ips, res.reason, res.content.decode("UTF-8")))
         sys.exit(1)
 
-    prod_data = json.loads(res.text)
+    prod_data = res.json()
```

Comment:
Is this compatible with python 3.4 which we have on SLE12 ?

Reply:
Yes, it is

Reply:
checked with the documentation and also tested (double check) that works for `Python 3.4.10`